pkg/tsdb/azuremonitor/loganalytics/azure-response-table-frame.go

```diff
@@ -45,12 +45,12 @@ func apiErrorToNotice(err *AzureLogAnalyticsAPIError) data.Notice {
 }
 
 // ResponseTableToFrame converts an AzureResponseTable to a data.Frame.
-func ResponseTableToFrame(table *types.AzureResponseTable, refID string, executedQuery string, queryType dataquery.AzureQueryType, resultFormat dataquery.ResultFormat) (*data.Frame, error) {
+func ResponseTableToFrame(table *types.AzureResponseTable, refID string, executedQuery string, queryType dataquery.AzureQueryType, resultFormat dataquery.ResultFormat, logLimitDisabled bool) (*data.Frame, error) {
```

Comment:
This is quite a long arguments list, I'd recommend considering if Functional Options would be a good fit here A plus would be that it'd allow you to set defaults for the options before you hit the private function `converterFrameForTable`.